sentry_sdk/tracing.py

```diff
@@ -606,6 +607,9 @@ def finish(self, hub=None):
             "spans": finished_spans,
         }
 
+        if hub.client.options["enable_profiling"]:
+            event["profile"] = self._profile.to_json()
```

Comment:
@szokeasaurusrex The profile should not be sent in the same envelope item as the transaction. The cocoa/android sdks currently create a separate envelope item containing just the profile and send that in the same envelope. One reason for this is in case we decide to separate the transaction and profile in separate envelopes.